src/components/Screens/SettingsScreen.jsx

```diff
@@ -65,10 +68,39 @@ const SettingsScreen = () => {
         }
     }, []);
 
+    const loadTerminal = useCallback(async () => {
```

Comment:
I think it is better to retrieve the store value from JS api unless you have a good reason.

Reply:
yeap, I changed this to use the store js api